src/auth/jwt-error.ts

```diff
@@ -0,0 +1,22 @@
+import express, { NextFunction } from "express";
+import { JwtError } from "@controllers/public-rest";
+
+// Handle TSOA errors
+export function jwtErrorHandler(
+  error: Error,
+  req: express.Request,
+  res: express.Response,
+  next: NextFunction
+) {
+  console.log("errorHandler::error", error);
```

Comment:
Same consideration regarding logging here:
https://github.com/badgeteam/badgehub-api/pull/87/files#r2083417572

Reply:
I'll remove (comment) the loggng